data/language/en-GB.txt

```diff
@@ -3676,6 +3676,7 @@ STR_6354    :{SMALLFONT}{BLACK}Medium density
 STR_6355    :{SMALLFONT}{BLACK}High density
 STR_6356    :{SMALLFONT}{BLACK}Spawns ducks if park contains water
 STR_6357    :{SMALLFONT}{BLACK}Removes all ducks from the map
+STR_6358    :A tile must have at least one tile element
```

Comment:
I'm not too keen on this new phrasing, as it now repeats the word 'tile'. What would be clearer? Not sure, myself.

Reply:
I think it's fine, "Tile" and "Tile element" are two separate things. In Dutch the latter would be one word.

Reply:
"A tile must have at least one element" looks fine to me, but I'm not too concerned with the repetition